src/app/page/Request/foodbox/ConfirmStep.jsx

```diff
@@ -53,16 +60,48 @@ function ConfirmStep({ dispatch, state }) {
     0
   );
 
-  function confirmRequest() {
-    //TODO create a new mission here based on customer details
-    if (isDonationRequest) {
-      //TODO set the funding status to not funded
-      //Send mission to firestore
+  // TODO we should try to create this mission before paying and if payment fails we delete it or something
+  async function confirmRequest() {
+    const { cart } = state;
+    let mission = {
+      type: MissionType.foodbox,
+      // what does this need to be??
+      // status: ???
```

Comment:
What does the status need to be for a new mission?

Reply:
new mission should have status 'Unassigned'  if its paid for by recipient it pops straight into 'funded' therefore 'tentative' (but I would do that *after* the payment is confirmed.)